docs/deployment/add-ons/load-balancer/README.md

```diff
@@ -0,0 +1,192 @@
+    - Following is a screenshot showing a certificate has been issued. Note: Status turns to `Issued` after few minutes of validation.
+        - ![successfully-issued-certificate](./files/successfully-issued-certificate.png)
+> - If you choose DNS validation for the validation of the certificates, you will be asked to create a CNAME type record in the hosted zone.
+    - Following is a screenshot of CNAME record of the certificate in `platform.example.com` hosted zone for DNS validation:
+            - ![DNS-record-for-certificate-validation](./files/DNS-record-for-certificate-validation.png)    
+
+1. Create a certificate for `*.example.com` in the region where your platform will be running
+1. Create a certificate for `*.platform.example.com` in the region where your platform will be running
+
+### Configure Ingress
+
+1. Export the ARN of the certificate created for `*.platform.example.com`:
+        1. `export certArn=<>`
+1. Configure the parameters for [ingress](../../../../awsconfigs/common/istio-ingress/overlays/https/params.env) with the certificate ARN of the subdomain
+    1. ```
+        printf 'certArn='$certArn'' > awsconfigs/common/istio-ingress/overlays/https/params.env
+        ```
+### Configure Load Balancer Controller
+
+Setup resources required for the load balancer controller:
+
+1. Make sure all the subnets(public and private) corresponding to the EKS cluster are tagged according to the `Prerequisites` section in this [document](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html). Ignore the requirement to have an existing ALB provisioned on the cluster. We will be deploying load balancer controller version 1.1.5 in the later section.
+    1. Specifically check if the following tags exist on the subnets:
+        1. `kubernetes.io/cluster/cluster-name` (replace `cluster-name` with your cluster name e.g. `kubernetes.io/cluster/my-k8s-cluster`). Add this tag in both private and public subnets. If you created the cluster using eksctl, you might be missing only this tag. Use the following command to tag all subnets by substituting the value of `TAG_VALUE` variable(`owned` or `shared`). Use `shared` as tag value if you have more than one cluster using the subnets:
+            - ```
+                export TAG_VALUE=<>
+                export CLUSTER_SUBNET_IDS=$(aws ec2 describe-subnets --region $CLUSTER_REGION --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$CLUSTER_NAME --output json | jq -r '.Subnets[].SubnetId')
+                for i in "${CLUSTER_SUBNET_IDS[@]}"
+                do
+                    aws ec2 create-tags --resources ${i} --tags Key=kubernetes.io/cluster/${CLUSTER_NAME},Value=${TAG_VALUE}
+                done
+                ```
+        1. `kubernetes.io/role/internal-elb`. Add this tag only to private subnet
+        1. `kubernetes.io/role/elb`. Add this tag only to public subnet.
```

Comment:
I think there are multiple private and public subnet, right ?
if so,
subnet -> subnets